gsd-web/package.json

```diff
@@ -11,6 +11,7 @@
     "build": "quasar build -m ssr",
     "heroku-postbuild": "yarn build",
     "dev": "quasar dev -m ssr",
+    "devinit": "npx jake setup",
```

Comment:
Would it be worth adding a guard clause to the init method to bail out early if the files already exist? Then we could merge the init into the `yarn dev` command, and reduce complexity for new devs starting out.

Reply:
So the way that `fs.copyFile` is configured (fs.constants.COPYFILE_EXCL). It will not overwrite the file if the destination already exists. I am happy to add a throw instead of logging the failure, which is the current behavior.

Reply:
Ah, I didn't realize that was how fs.copyFile works. It sounds like it's already doing exactly the behavior I was looking for.

In that case, we can safely simplify to:
```json
  "dev": "npx jake setup && quasar dev -m ssr",
```